package.json

```diff
@@ -13,20 +13,28 @@
     "test": "jest --notify --config=jest.config.js"
   },
   "dependencies": {
-    "@babel/core": "^7.12.10",
-    "@babel/preset-env": "^7.12.11",
     "@testing-library/jest-dom": "^5.11.9",
     "babel-jest": "^26.6.3",
     "gatsby": "^2.26.1",
     "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "save": "^2.4.0"
+    "react-dom": "^16.12.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.10",
```

Comment:
[C] If you run your dev server with ts-node or ts-node-dev (which has a watch option) you can get ride of babel altogether.

Reply:
I've removed it. There was a heap of testing config that I had to setup which is specific for gatsby